uni/lib/view/Widgets/schedule_row.dart

```diff
@@ -3,30 +3,39 @@ import 'package:uni/view/Widgets/schedule_event_rectangle.dart';
 import 'package:uni/view/Widgets/schedule_time_interval.dart';
 import 'package:add_2_calendar/add_2_calendar.dart';
 import 'package:material_design_icons_flutter/material_design_icons_flutter.dart';
+import 'package:uni/controller/local_storage/app_shared_preferences.dart';
+import 'package:uni/model/entities/exam.dart';
 
-class ScheduleRow extends StatelessWidget {
-  final String subject;
-  final List<String> rooms;
-  final String begin;
-  final String end;
-  final DateTime date;
-  final String teacher;
-  final String type;
+class ScheduleRow extends StatefulWidget {
```

Comment:
There was a issue created to change this name, as it should be ExamRow.
Created another issue to ease with merging as the overall structure is changing and it could be  troublesome. Also because it's used in testing and it's totally out of scope of this issue